include/dev/evdev.h

```diff
@@ -82,7 +93,14 @@ uint16_t evdev_scancode2key(int *statep, int scancode);
 
 /* A wrapper for the evdev_support_key function - marks all the AT-compatible
  * keys as supported. */
-void evdev_support_all_known_keys(evdev_dev_t *evdev);
+void evdev_atkbd_support_all_known_keys(evdev_dev_t *evdev);
+
+/* Convert a USB mouse button number to an evdev-compatible keycode. */
+uint16_t evdev_hid2btn(uint8_t btnum);
+
+/* A wrapper for the evdev_support_key function -
+ * marks all the supported mouse keys. */
+void evdev_support_all_known_buttons(evdev_dev_t *evdev);
```

Comment:
```suggestion
void evdev_m_support_all_known_buttons(evdev_dev_t *evdev);
```
to stay consistent with other function names, not sure if there should be an abbreviation before m.

Reply:
`evdev_support_all_mouse_buttons`?

Reply:
`evdev_support_all_mouse_buttons` it is.